src/discussions/posts/data/__factories__/threads.factory.js

```diff
@@ -2,13 +2,15 @@ import { Factory } from 'rosie';
 
 Factory.define('thread')
   .sequence('id', (idx) => `thread-${idx}`)
-  .sequence('title', ['topic_id'], (idx, topicId) => `This is Thread-${idx} in topic ${topicId}`)
+  .sequence('title', ['topic_id', 'author'], (idx, topicId, author) => `This is Thread-${idx} in topic ${topicId} by ${author}`)
```

Comment:
You did not need to do this, this can break a lot of other tests. Just override the title in your own use case.

Reply:
You can just build a threadlist with overridden title as follows:

` Factory.build('threadsResult', null, { threadAttrs: { author: 'myauthor', title: 'post by myauthor' } });`

This will generate multiple threads with the same name, but that's OK.